spec/features/admin_creates_quiz_spec.rb

```diff
@@ -8,10 +8,31 @@
     click_on "New Quiz"
   end
 
-  scenario "successful quiz creation" do
+  scenario "successful quiz creation", js: true do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.